### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by vberger**
This is basic translation of nheko for french. There is likely room for improvement of several wordings, but this make things correctly understandable for a french user.

### CODE REVIEW

File: resources/langs/nheko_fr.ts [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+    <name>LoginPage</name>
+    <message>
+        <location filename="../../src/LoginPage.cc" line="+79"/>
+        <source>Matrix ID</source>
+        <translation>Identifiant Matrix</translation>
+    </message>
+    <message>
+        <location line="+1"/>
+        <source>e.g @joe:matrix.org</source>
+        <translation>ex: @joe:matrix.org</translation>
```
Comment (erdnaxeli) [inline resources/langs/nheko_fr.ts]: A non-breaking space is required before double punctuation in french.

File: resources/langs/nheko_fr.ts [hunk 2]
```diff
@@ -0,0 +1,583 @@
...
+    </message>
+    <message>
+        <location line="+6"/>
+        <source>An unknown error occured. Please try again.</source>
+        <translation>Une erreur inconnue s&apos;est produite. Veuillez essayer à nouveau.</translation>
+    </message>
+    <message>
+        <location line="+23"/>
+        <source>Malformed response. Possibly not a Matrix server</source>
+        <translation>La réponse du serveur est malformée. il est possible qu&apos;il ne s&apos;agisse pas d&apos;un serveur Matrix</translation>
```
Comment (erdnaxeli) [inline resources/langs/nheko_fr.ts]: Missing uppercase to "il".

File: resources/langs/nheko_fr.ts [hunk 3]
```diff
@@ -0,0 +1,583 @@
...
+        <source>SHOW MORE</source>
+        <translation>MONTRER PLUS</translation>
+    </message>
+</context>
+<context>
+    <name>QuickSwitcher</name>
+    <message>
+        <location filename="../../src/QuickSwitcher.cc" line="+70"/>
+        <source>Search for a room...</source>
+        <translation>Chercher un salon...</translation>
```
Comment (erdnaxeli) [inline resources/langs/nheko_fr.ts]: There is the following character in french: "…".

File: resources/langs/nheko_fr.ts [hunk 4]
```diff
@@ -0,0 +1,583 @@
...
+    <name>dialogs::Logout</name>
+    <message>
+        <location filename="../../src/dialogs/Logout.cc" line="+47"/>
+        <source>CANCEL</source>
+        <translation>ANNULER</translation>
+    </message>
+    <message>
+        <location line="+10"/>
+        <source>Logout. Are you sure?</source>
+        <translation>Déconnexion. Êtes-vous sûr·e ?</translation>
```
Comment (erdnaxeli) [inline resources/langs/nheko_fr.ts]: Really? :p

File: resources/langs/nheko_fr.ts [hunk 5]
```diff
@@ -0,0 +1,583 @@
...
+        <source>Cancel</source>
+        <translation>Annuler</translation>
+    </message>
+    <message>
+        <location line="+72"/>
+        <source>Media type: %1
+Media size: %2
+</source>
+        <translation>Type de média: %1
+Taille du média: %2
```
Comment (erdnaxeli) [inline resources/langs/nheko_fr.ts]: Non-breaking space

**Comment by mujx** [comment]
Thanks a lot @vberger! 

I don't really know French so I will have to leave this one up to the French-speaking contributors.

File: resources/langs/nheko_fr.ts (hunk 4 above)
Comment (vberger) [reply resources/langs/nheko_fr.ts]: What's your point?

File: resources/langs/nheko_fr.ts [hunk 6]
```diff
@@ -0,0 +1,583 @@
...
+        <source>Join a room</source>
+        <translation>Rejoindre un salon</translation>
+    </message>
+</context>
+<context>
+    <name>TextInputWidget</name>
+    <message>
+        <location filename="../../src/TextInputWidget.cc" line="+445"/>
+        <source>Write a message...</source>
+        <translation>Écrivez un message...</translation>
```
[the remark this answers is not in the document]
Comment (vberger) [reply resources/langs/nheko_fr.ts]: C'est le placeholder du champ dans lequel on écrit le message, du coup ça me semblait plus pertinent d'utiliser "Écrivez...". Contrairement à si c'était un bouton où là je suis d'accord "Écrire..." aurait été plus adapté.

**Pull request merged**